lms/envs/common.py

```diff
@@ -2746,7 +2746,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
     # Verified Certificates
     'WHAT_IS_VERIFIED_CERT': None,
-    'OUR_TEAM': 'our_team',
```

Comment:
If this change was already added in core changes, please remove it from there.

Reply:
I'll take a look and update it. Thanks for pointing out!